source/develop/release-management/features/storage/incremental-backup.html.md

```diff
@@ -317,17 +318,57 @@ XXX Add disk attachment incremental setting.
 XXX Add checkpoints table
 
 - New ```vm_backups``` table
-  - fields: id, vm_id, incremental, creation_date, status
+  - fields: id, vm_id, checkpoint_id, creation_date, status
   - status: initializing, ready, transferring, finallizing
 - ```images``` table
-  new field: ```backup_id``` (foreign key ```bakcups:id```)
+  new field: ```backup_id``` (foreign key ```vm_backups:id```)
   new field: ```backup_url``` (nbd url used in image transfer ticket)
+  new field: ```checkpoint_id``` (foreign key ```image_checkpoint:id```)
+- New ```image_checkpoint``` table
+  - fileds: id, parent, creation_date
+
 
 ### REST
 
+#### Enabling backup for VM disk
+
+Specify 'backup' flag on 'disk_attachment' entity.
+
+Request:
+```
+POST /vms/vm-uuid/diskattachments
+```
+
+Response:
+```
+<disk_attachment>
+    ...
+    <backup>false|true</backup>
```

Comment:
The meaning of backup=false and backup=true is not clear.

I think we need:
```
backup=incremental
backup=full
```
And maybe later we will want to add:
```
backup=none
```
To disable backup for a disk. For example, maybe the disk is  a scratch disk and the user does not want to backup it contents.